crates/dojo-lang/src/compiler.rs

```diff
@@ -102,9 +102,22 @@ impl Compiler for DojoCompiler {
         for (decl, class) in zip(contracts, classes) {
             let target_name = &unit.target().name;
             let contract_name = decl.submodule_id.name(db.upcast_mut());
-            let file_name = format!("{target_name}-{contract_name}.json");
+            let contract_name_snake = contract_name.to_case(Case::Snake);
 
-            let mut file = target_dir.open_rw(file_name.clone(), "output file", ws.config())?;
+            let is_model = match &class.abi {
+                Some(abi) => abi.items.iter().any(|i| match i {
```

Comment:
Recent update to Cairo 2.3.0 seems to have made `items` private which invalidates this approach.

Reply:
Added a solution here: https://github.com/starkware-libs/cairo/pull/4310